README.md

```diff
@@ -794,6 +794,24 @@
 
     ```
 
+  - Remove all trailing whitespace
+
+    ```javascript
+    // bad
+    function() {∙∙
+      var name;
+    ∙∙
+      // ... stuff ...
+    }
+
+    // good
+    function() {
+      var name;
+
+      // ... stuff ...
+    }
+    ```
+
```

Comment:
Yeah sublime is 
Preferences / Settings - User" to add the following:
`{ "trim_trailing_white_space_on_save": true }`

IntelliJ under preferences, editor
https://i.imgur.com/JDKfldN.png

JSCS has autofix (`jscs src --fix`) as of 1.12 and working on applying it to all rules.